plugins/versionpress/bootstrap.php

```diff
@@ -54,6 +54,13 @@
     define('VERSIONPRESS_REQUIRE_API_AUTH', true);
 }
 
+if (!defined('VP_COMMIT_MESSAGE_PREFIX')) {
```

Comment:
We use prefix `VP_` for public constants, `VERSIONPRESS_` for internal constants. I'm not sure if this should be a public constant.